plugins/modules/net_tools/nmcli.py

```diff
@@ -714,7 +714,7 @@ def connection_options(self, detect_change=False):
             })
 
         # Layer 2 options.
-        if self.mac_conn_type:
+        if self.mac:
```

Comment:
I think you also need to update the documentation, which still claims that `mac` is only used for `type=bridge`.